docs/README.md

```diff
@@ -226,9 +226,9 @@ Daha fazla bilgi için [**Mahremiyet**](https://guvendekal.org/#/mahremiyet) ve
         </div>
     </div>
     <div class="service-box">
-        <img src="docs/images/vaultwarden.png" alt="icon">
+        <img src="docs/images/proton-pass.png" alt="icon">
         <div class="title">
-            <a href="https://guvendekal.org/#/sifre-yoneticileri?id=-vaultwarden" target="_blank" style="color: inherit; text-decoration: none;">Vaultwarden</a>
```

Comment:
Vaultwarden neden kaldırıldı? Acaba satır eklenmek istenirken yanlışıkla mı silindi?

Reply:
Vaultwarden, sunucu kurmaya yaradığından kaldırıldı. İleride self-host rehberi açarsak oraya ekleyebiliriz ancak Keepass, Bitwarden ve Proton Pass'in yanında farklı kategoride bir uygulama olarak kalıyordu. Örneğin Vaultwarden ile  kurduğun sunucuya Bitwarden ile bağlanıyorsun yada başka istemcilerle.

Reply:
Vaultwarden bir Bitwarden forku değil miydi? 🤔 ben mi yanlış hatırlıyorum. Hatta kendi sunucumda da var. Basit bir docker container'ına kurup bitwarden ile bağlayabiliyoruz. Sunucu kurmaktan kastınız nedir tam olarak?

Reply:
Benim hatırladığım kadarıyla Bitwarden forku ancak Rust ile tekrar yazılmıştı sanırım. Sonuç olarak bir istemci değil bildiğim kadarıyla şifrelerin depolanmasına yarayan bir sunucu işlevi gördüğünü hatırlıyorum. Ben eklemiştim Vaultwarden'i ancak ne işe yaradığını tam hatırlamıyorum.

Reply:
Tamam şimdi anladım durumu. Sunucuya yüklediğimiz anlam farklı.

Reply:
Aynen aynen ben biraz yanlış anlattım sanırım. İşteyim ondan tam kafamı kullanamıyorum. Vaultwarden, Bitwarden sunucularını kullanmak istemeyen (kendi şifre sunucusunu kurmak isteyen) kullanıcılara 3. parti bir sunucu sağlıyor.